core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/metadata/OperatorGroupConstants.scala

```diff
@@ -32,7 +33,14 @@ object OperatorGroupConstants {
       CLEANING_GROUP,
       List(GroupInfo(JOIN_GROUP), GroupInfo(AGGREGATE_GROUP), GroupInfo(SORT_GROUP))
     ),
-    GroupInfo(MACHINE_LEARNING_GROUP, List(GroupInfo(SKLEARN_GROUP), GroupInfo(HUGGINGFACE_GROUP))),
+    GroupInfo(
+      MACHINE_LEARNING_GROUP,
+      List(
+        GroupInfo(SKLEARN_GROUP),
+        GroupInfo(HUGGINGFACE_GROUP),
+        GroupInfo(MODEL_PERFORMANCE_GROUP)
```

Comment:
maybe call it Machine Learning General group and put the split, scorer in it?

Reply:
fix